features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/assets/default/asset.jag

```diff
@@ -47,9 +47,23 @@ require('/modules/publisher.js').exec(function(ctx) {
     var filterManager = publisher.filterManager;
 
     mod = jagg.module("documentation");
-
     var typeList = apiProvider.getEnabledAssetTypeList();
 
+    function getOwnerList() {
+        var businessOwnerList = apiProvider.getBusinessOwnerList();
+        var ownersList = JSON.parse(stringify(businessOwnerList));
+        var ownerNames = "";
+        for (var i = 0; i < ownersList.length; i++) {
```

Comment:
Remove this string manipulation and use JSON Map to maintain business owner extra details